### DISCUSSION TITLE
How to suppress Photos and Videos from DLNA

### DISCUSSION THREAD

**Discussion by millerabel** [LMS-Community/slimserver#659] [Q&A] [answered]
Hi All, I've configured LMS on my iMac to serve a copy of iTunes music library to a Cambridge Audio CXNv2. However, I don't want to publish any photos or videos from this server. I have unchecked Videos and Photos columns in Basic Settings next to the sole music folder. There is a blank line at bottom of the Folder list that has Music, Video, and Photos columns checked. I can't uncheck them ... each time I try, then Save Settings, the columns for this blank folder line are re-checked.

The Cambridge Audio StreamMagic app always shows empty Photos and Videos folders. The folders are apparently advertised by LMS even when there are no discovered photos and videos in any of the folders and even when Photos and Videos columns are unchecked.

Is there some way to suppress these empty virtual folders?

Many thanks!

**Comment by mherger** [comment, marked as answer]
I don't think you can hide those items.

**Comment by millerabel** [comment]
Thanks Michael, do you know if this is part of the DLNA media server specification or is it a design choice in the LMS?

— Miller

On Sep 13, 2021, at 12:05 AM, Michael Herger ***@***.***> wrote:

﻿
I don't think you can hide those items.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by michaelherger** [comment]
I believe it's a LMS thing.
